packages/frontend/package.json

```
@@ -1,25 +1,35 @@
 {
   "name": "eslint-config-trybe-frontend",
-  "version": "1.2.3",
+  "version": "1.3.0",
   "description": "ESLint shared config used by Trybe on frontend projects",
   "main": "config.json",
   "keywords": [
     "eslint",
     "trybe",
     "frontend"
   ],
-  "author": "Roz <roz@rjmunhoz.me>",
+  "author": "Equipe Front",
```

```suggestion
  "author": "Equipe Front Trybe",
```

Vale adicionar o nome da Trybe aqui também?